hw/top_earlgrey/doc/_index.md

```diff
@@ -325,45 +277,26 @@ A security section will follow later to show how these work together.
 
 ##### AES
 
-[AES](https://nvlpubs.nist.gov/nistpubs/fips/nist.fips.197.pdf)
-is the primary
-[symmetric encryption](https://en.wikipedia.org/wiki/Symmetric-key_algorithm)
-and decryption mechanism used in OpenTitan protocols.
+[AES](https://nvlpubs.nist.gov/nistpubs/fips/nist.fips.197.pdf) is the primary [symmetric encryption](https://en.wikipedia.org/wiki/Symmetric-key_algorithm) and decryption mechanism used in OpenTitan protocols.
 AES runs with key sizes of 128b, 192b, or 256b.
```

Comment:
do you think it's worth mentioning GCM at all? I guess we don't really do anything to natively support it?

Reply:
Done. Added a note to mentioned that this is not supported by hardware but that it can be implemented with some software support.